winforms/src/toga_winforms/libs/WebView2/README.md

```diff
@@ -1,9 +1,12 @@
 # WebView2
 
-These DLLs are the official stable release v1.0.1150.38, downloaded from
+These DLLs are the official stable release v1.0.2592.51 (net462), downloaded from
 [NuGet](https://www.nuget.org/packages/Microsoft.Web.WebView2). They are
 released under the terms of the [license in this folder](./LICENSE.txt).
 
+To make them work after download, the "mark of the web" (MotW) had been set manually.
+See also https://github.com/beeware/briefcase/issues/1890
+
```

Comment:
I'm not 100% certain this is required.

If pip install toga into a local virtual environment will work, then any MotW issues aren't Toga issues - they're entirely packaging/Briefcase issues.

```suggestion
```